kpops/components/streams_bootstrap_v3/base.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+import logging
+from abc import ABC
+from typing import TYPE_CHECKING, Any
+
+import pydantic
+from pydantic import AliasChoices, ConfigDict, Field
+
+from kpops.component_handlers.helm_wrapper.model import HelmRepoConfig
+from kpops.components.base_components import KafkaApp
+from kpops.components.base_components.helm_app import HelmApp, HelmAppValues
+from kpops.components.common.topic import KafkaTopic, KafkaTopicStr
+from kpops.utils.docstring import describe_attr
+from kpops.utils.pydantic import (
+    CamelCaseConfigModel,
+    DescConfigModel,
+    exclude_by_value,
+    exclude_defaults,
+)
+
+if TYPE_CHECKING:
+    try:
+        from typing import Self  # pyright: ignore[reportAttributeAccessIssue]
+    except ImportError:
+        from typing_extensions import Self
+
+
+STREAMS_BOOTSTRAP_HELM_REPO = HelmRepoConfig(
+    repository_name="bakdata-streams-bootstrap",
+    url="https://bakdata.github.io/streams-bootstrap/",
+)
+STREAMS_BOOTSTRAP_VERSION = "3.0.0"
+
+# Source of the pattern: https://kubernetes.io/docs/concepts/containers/images/#image-names
+IMAGE_TAG_PATTERN = r"^[a-zA-Z0-9_][a-zA-Z0-9._-]{0,127}$"
+
+log = logging.getLogger("StreamsBootstrapV3")
+
+
+class StreamsBootstrapV3Values(HelmAppValues):
```

Comment:
this should also have a field kafka with type KafkaConfig to centrally configure bootstrapservers and sr